content/contact/_index.en.md

```diff
@@ -0,0 +1,109 @@
+70180 Stuttgart
+</address>
+
+The association is registered at the Stuttgart Local Court under association 
+registration number VR 725447.
+
+The board represents the association. Chairs of the board are:
+
+* Thomas Obst, first chairman
+* Reinhard Otter, second chairman
+* Margit Obst, treasurer
+
+The board can be reached at <script type="text/javascript"><!--/* Generated by www.email-encoder.com */
+for(var dcibss=["dA","QA","cw","Og","bQ","ZQ","PQ","YQ","ZQ","aQ","Lg","LQ","cw","bA","ZA","cA","bw","ZQ","aA","aQ","cg","cg","bg","dA","cw","Zg","YQ","Pg","aQ","aw","ZQ","PA","bg","bg","IA","aQ","YQ","aw","cA","bw","QA","bA","bg","Zw","cw","YQ","dg","aQ","Yg","ZQ","bw","cw","cw","Lw","Ig","cg","bw","Ig","ZA","bg","ZQ","Lg","Pg","PQ","bA","aw","cg","Yw","cw","ZQ","cg","Ig","Ig","Yg","dg","bg","bw","bw","YQ","bw","bA","cg","dA","IA","YQ","bg","Zw","bw","bw","cw","cg","bQ","YQ","PA"],pbcshz=[67,24,66,15,9,81,50,21,79,58,39,56,19,46,23,73,17,74,3,54,85,65,59,20,33,6,1,93,30,60,34,90,69,22,44,77,10,31,26,64,71,57,75,89,80,92,63,11,29,32,37,49,83,91,8,4,14,61,70,28,5,86,62,7,12,78,88,45,48,27,18,51,43,76,16,35,72,25,47,84,55,41,13,2,68,82,42,87,40,36,38,52,53,0],tftzqr=new Array,i=0;i<pbcshz.length;i++)tftzqr[pbcshz[i]]=dcibss[i];for(var i=0;i<tftzqr.length;i++)document.write(atob(tftzqr[i]+"=="));
+// --></script><noscript>Please enable JavaScript to see the email address (<a href="https://www.email-encoder.com/enablejs/" target="_blank" rel="noopener noreferrer">How-to</a>).</noscript>.
+
+Thomas Obst is responsible for the editorial content of this website and can 
+be reached as chairman of the association board via the above-mentioned 
+association address.
+
+### Contact by e-mail
+
+If you are at a loss, you can also send general inquiries to <script 
+type="text/javascript"><!--/* Generated by www.email-encoder.com */
+for(var vfawqb=["cg","bg","ZQ","cg","LQ","cg","aQ","bA","YQ","Lg","bw","Lg",
+"ZQ","bg","Zg","aw","cw","bw","ZQ","cw","ZQ","bw","QA","cg","cw","cg","Ig",
+"bg","Zg","Zw","bw","bQ","bw", "ZQ","Yg","PA","PQ","YQ","IA","YQ","Ig","aw",
+"QA","bw","cw","Pg","cA","bA","dA","Pg","ZQ","Yg","cw","bg","ZQ","Lw","bg",
+"aQ","PA","aA","bQ","bw","Zg","YQ","Ig","cw","YQ","aQ","aQ", "aQ","Zw","bA",
+"aQ","bw","IA","bg","PQ","Ig","aQ","bg","aw","Og","bA","bw","Yw","cA"],
+sjkvcl=[77,24,73,4,52,34,26,51,1,35,33,78,5,60,61,56,75,14,66,45,28,79,20,80,
+29,37,39,67,18,81, 21,9,76,30,25,0,46,84,40,49,8,70,63,64,44,85,65,12,13,58,
+23,68,32,31,71,83,17,50,82,3,48,36,6,43,57,72,10,54,59,16,38,42,11,19,2,74,7,
+47,69,55,27,15,53,62,41,22],qgcwtm=new Array,i=0; i<sjkvcl. length;i++)
+qgcwtm[sjkvcl[i]]=vfawqb[i];for(var i=0;i<qgcwtm.length;i++)document.write
+(atob(qgcwtm[i]+"=="));
+// --></script><noscript>Please enable JavaScript to see the email address 
+(<a href="https://www.email-encoder.com/enablejs/" target="_blank" 
+rel="noopener noreferrer">How-to</a>).</noscript>. However, please try your 
+luck with the community first.
```

Comment:
Please do not duplicate these contents in English, instead, refer to the German original page for the legally required information.

Reply:
Ok. I figured it would still be valuable to have such versions as I think there is value in knowing who's responsible for what, and referring to the German version limits the site to those that understand German. If you'd rather not have the English version, I'll just remove it.